src/notation/qml/MuseScore/NotationScene/NoteInputBarCustomiseDialog.qml

```diff
@@ -116,6 +116,8 @@ QmlDialog {
 
                 model: customiseModel
 
+                maximumFlickVelocity: 1000
```

Comment:
I guess we can specify some default value for "ui.theme" or even for StyledScrollBar component and use it here

@Eism since it's ur code, what would you prefer? :)